index.html

```diff
@@ -1,9 +1,21 @@
 <html>
+    <head>
+   <link rel="stylesheet" href="style.css">
+   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    </head>
     <body>
-        <div>
-            <h1>
-                hi  this is rishi
-            </h1>
+        <h2 style="text-align: start;">E-Safety</h2>
+    <div class="center">
+        <div class="color-box">
+            <br><br>
+        <h2>Login</h2>
+        <br>
+        <input type="text" placeholder="Email Or Username" style="width: 50%;"> <br><br>
```

Comment:
) Same comment as above, put styling in separate css file
) Dont use <br> tags for next line, you should be using margin css attibute
Ex:
<div class="name-you-give">
   <input type="text" placeholder="Email Or Username" >
</div>